actionpack/lib/action_dispatch/middleware/templates/rescues/missing_exact_template.html.erb

```diff
@@ -0,0 +1,23 @@
+<header>
+  <h1>No template for interactive request</h1>
+</header>
+
+<div id="container">
+  <h3><%= h @exception.message %></h3>
+
+  <p class="summary">
+    <strong>NOTE!</strong><br>
+    Unless told otherwise, Rails expects an action to render a template with the same name,<br>
+    contained in a folder named after its controller.
+
+    If this controller is an API responding with 204 (No Content), <br>
+    which does not require a template,
+    then this error will occur when trying to access it via browser,<br>
+    since we expect an HTML template
+    to be rendered for such requests. If that's the case, carry on.
+  </p>
+
+  <%= render template: "rescues/_source" %>
```

Comment:
Not sure if the source and the trace is going to be useful for this exception. The source is always inside Rails. I'd also remove the request_and_response. This page is more informative than actually an application error.